Source/Core/Common/SysConf.cpp

```diff
@@ -430,11 +430,10 @@ void SysConf::UpdateLocation()
   // Clear the old filename and set the default filename to the new user path
   // So that it can be generated if the file does not exist in the new location
   m_Filename.clear();
-  // Note: We don't use the dummy Wii root here (if in use) because this is
-  // all tied up with the configuration code.  In the future this should
-  // probably just be synced with the other settings.
+  // In the future the SYSCONF should probably just be synced with the other settings.
   m_FilenameDefault =
-      File::GetUserPath(D_WIIROOT_IDX) + DIR_SEP WII_SYSCONF_DIR DIR_SEP WII_SYSCONF;
+      File::GetUserPath(root_type == WiiRootType::User ? D_WIIROOT_IDX : D_SESSION_WIIROOT_IDX) +
```

Comment:
You should probably make it possible to call the following function from here: https://github.com/dolphin-emu/dolphin/blob/3cfc49a613a0f8e137c251694c97f0f3589f3216/Source/Core/Common/NandPaths.cpp#L62